langlib/lang.__internal/src/main/java/org/ballerinalang/langlib/internal/GetElementType.java

```diff
@@ -36,13 +38,18 @@
 
     public static BTypedesc getElementType(Object td) {
         BTypedesc bTypedesc = (BTypedesc) td;
-        Type type = bTypedesc.getDescribingType();
+        return getTypeDescValue(bTypedesc.getDescribingType());
+    }
+
+    private static BTypedesc getTypeDescValue(Type type) {
```

Comment:
Should this be something like `getElementTypeDescValue`? Since the "element" type extraction also happens here?

Reply:
Requested changes in https://github.com/ballerina-platform/ballerina-lang/pull/32257